Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -140,14 +140,14 @@ class SentryFileManagerTests: XCTestCase {
     }
     
     func testStoreAndReadCurrentSession() {
-        let expectedSession = SentrySession()
+        let expectedSession = SentrySession(releaseName: "")
         sut.storeCurrentSession(expectedSession)
         let actualSession = sut.readCurrentSession()
         XCTAssertTrue(expectedSession.distinctId == actualSession?.distinctId)
     }
 
     func testStoreDeleteCurrentSession() {
-        sut.storeCurrentSession(SentrySession())
+        sut.storeCurrentSession(SentrySession(releaseName: ""))
```

Comment:
L: `initWithReleaseName` was a good catch. But made me realize that maybe we should validate not to be an empty string and change this tests.